crates/astria-conductor/src/driver.rs

```diff
@@ -102,14 +124,14 @@ impl Driver {
         info!("Starting driver event loop.");
         loop {
             select! {
-                res = self.network.0.next() => {
+                res = self.network.get_mut().0.next() => {
                     if let Some(res) = res {
-                        self.handle_network_event(res)?;
+                        self.handle_network_event(res).await.wrap_err("failed to handle network event")?;
```

Comment:
In the case of an error, should the entire event loop abort? Or should this just log the error and continue the loop?

Reply:
for the gossip network errors, we should continue (ie. we receive some invalid data and can't decode it)

it seems if we fail on `handle_driver_command`, we should probably abort. should I update here?